main.py

```diff
@@ -15,6 +15,8 @@
 parser_train.add_argument("-o", "--output", type=str, required=True, help="path to save model to")
 parser_train.add_argument("-c", "--config", type=str, help="config with training parameters")
 parser_train.add_argument("-d", "--data", type=str, default="./data", help="directory with nuScenes dataset")
+parser_train.add_argument("-g", "--gpu", type=int, nargs='+', default="0", help="list of available gpus")
```

Comment:
Я не уверен, но `nargs='+', default="0"` вроде будет выдавать ошибку, если не указывать `-g`
```suggestion
parser_train.add_argument("-g", "--gpu", type=int, nargs="*" default="0", help="list of available gpus")
```